AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -166,7 +232,16 @@ public final class UPIComponent: PaymentComponent,
         qrCodeGenerationLabelContainerItem.isHidden.wrappedValue = true
         formViewController.append(FormSpacerItem(numberOfSpaces: 1))
         formViewController.append(qrCodeGenerationLabelContainerItem)
-        formViewController.append(virtualPaymentAddressItem)
+
+        if !upiPaymentMethod.apps.isEmpty {
+            for item in upiAppsList {
+                formViewController.append(item)
+            }
+            formViewController.append(virtualPaymentAddressItem)
+            virtualPaymentAddressItem.isHidden.wrappedValue = true
+        } else {
+            formViewController.append(virtualPaymentAddressItem)
+        }
```

Comment:
```suggestion
        formViewController.append(virtualPaymentAddressItem)
        
        if !upiPaymentMethod.apps.isEmpty {
            for item in upiAppsList {
                formViewController.append(item)
            }
            virtualPaymentAddressItem.isHidden.wrappedValue = true
        }
```

Do we need to add `virtualPaymentAddressItem` but hide it? Can we not add it?

Reply:
this will add the `virtualPaymentAddressItem` to the top of the list which is not right according to the UI.

Reply:
fair, then it can be after IF